isis/tests/FunctionalTestsJitterfit.cpp

```diff
@@ -0,0 +1,135 @@
+#include "CSVReader.h"
+#include "Fixtures.h"
+#include "jitterfit.h"
+#include "TestUtilities.h"
+
+#include "gmock/gmock.h"
+#include <fstream>
+
+using namespace Isis;
+
+static QString APP_XML = FileName("$ISISROOT/bin/xml/jitterfit.xml").expanded();
+TEST_F(TempTestingFiles, FunctionalTestJitterfitDefault){
+  QTemporaryDir prefix;
+  QString outputCoeffs = prefix.path() + "/coef.csv";
+  QString outputResiduals = prefix.path() + "/residuals.csv";
+  QString outputRegistrationStats = prefix.path() + "/regStats.pvl";
+
+  // Copy input data to tempdir because jitterfit writes to cube labels
+  QString dataCube = prefix.path() + "/source.cub";
```

Comment:
This section would be easier to read if all of the main cube was processed and then the check line cube instead of interleaved